src/components/e-invoice/PaymentMeans.tsx

```diff
@@ -380,7 +378,7 @@ export const PaymentMeans = forwardRef<PaymentMeansFormComponent, Props>(
         ? PAYMENT_MEANS_FORM_ELEMENTS[
             form.values.payment_means[0]
               .code as keyof typeof PAYMENT_MEANS_FORM_ELEMENTS
-          ]
+          ] || []
```

Comment:
@beganovich When the component is loading for the first time, I experienced page breaking because the `code` prop had a value that did not match any elements from the array. As a result, the elements variable was `undefined` and caused the page to break, so I added a fallback to an empty array for any edge cases.

Reply:
@Civolilah we will need to set a default of "1" for the code please if none is present.

![image](https://github.com/user-attachments/assets/1862cac9-9920-442e-85a2-066969e3c94e)

Otherwise user will never be able to save

Reply:
@turbo124 Sure thing, the code has been set to `'1'` by default if nothing is populated, and I also disabled the possibility of clearing the value (setting to empty string) for this selector so we can avoid having an empty string as a value from the React side. Let me know your thoughts.